articles/install-guide/index.md

```diff
@@ -1,53 +1,225 @@
-> [!div class="button"]
-> [Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit)
+1. Install the `qsharp` package
 
-Then, check our [quickstart guide](xref:microsoft.quantum.write-program?tabs=tabid-vs2019) for step-by-step instructions for your first quantum program.
+    ```bash
+    conda install -c quantum-engineering qsharp
+    ```
 
-### Visual Studio Code ###
+1. Install the `iqsharp` package
 
- If you are already a Visual Studio Code user, start by installing the latest version of the [.NET Core SDK](https://dotnet.microsoft.com/) (2.2 or later) by following the instructions from the [.NET downloads page](https://www.microsoft.com/net/download). Then, you can install the extension to get started developing Q# in Visual Studio Code.
-> [!div class="button"]
-> [Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode)
+   ```bash
+   conda install -c quantum-engineering iqsharp
+   ```
 
-Then, check our [quickstart guide](xref:microsoft.quantum.write-program?tabs=tabid-vscode) for step-by-step instructions for your first quantum program.
+1. Verify the installation by creating a `Hello World` application
 
+    - Create a minimal Q# operation, by creating a file called `Operation.qs`, and adding the following code to it:
 
-### Python ###
+        ```qsharp
+        namespace HelloWorld
+        {
+            open Microsoft.Quantum.Intrinsic;
+            open Microsoft.Quantum.Canon;
 
-If you are a Python user, check out our [getting started with Python and the Quantum Development Kit guide](xref:microsoft.quantum.install.python).
+            operation HelloQ() : Result
+            {
+                Message($"Hello from quantum world!");
+                return Zero;
+            }
+        }
+        ```
 
+    - Create a python program called `run_hello_q.py` to call the Q# `HelloQ()` operation:
 
-## Learn Quantum Computing with Q# ##
+        ```python
+        from HelloWorld import HelloQ
 
-* [Quantum Katas](https://github.com/Microsoft/QuantumKatas)
-* [Samples](https://github.com/Microsoft/Quantum)
-* [Quantum Computing Concepts](xref:microsoft.quantum.concepts.intro)
-* [Microsoft Quantum Blog](https://cloudblogs.microsoft.com/quantum/?ext)
-* [Q# Blog](https://devblogs.microsoft.com/qsharp/)
+        HelloQ.simulate()
+        ```
 
+    - Run the program:
 
+        ```bash
+        python run_hello_q.py
+        ```
+
+    - Verify the output. Your program should output the following lines:
+
+        ```bash
+        Hello from quantum world!
+       0
+       ```
+
+## Develop with Jupyter notebooks
+
+1. Pre-requisites
+
+    - [Python](https://www.python.org/downloads/) 3.6 or later
+    - [Jupyter notebooks](https://jupyter.readthedocs.io/en/latest/install.html)
```

Comment:
```suggestion
    - [Jupyter Notebook](https://jupyter.readthedocs.io/en/latest/install.html)
```

Since this is in the portion of the workflow that assumes `conda` is present, is it worth mentioning that the Jupyter Notebook client can be installed with `conda install notebook`?

Reply:
Let's just do one mode. Which do you think is best/easiest?